egasub/ega/entities/analysis.py

```diff
@@ -1,7 +1,7 @@
 
 class Analysis(object):
     def __init__(self, title, description, study_id, sample_references, analysis_center, analysis_date,
-                 analysis_type_id, files, attributes, genome_id, chromosome_references, experiment_type_id,platform):
+                 analysis_type_id, files, attributes, genome_id, chromosome_references, experiment_type_id,platform,alias):
```

Comment:
I didn't mention it before, as parameters are position based, I think we should put `alias` in the first position (after `self`).

Same for other classes.